pipeline_documentation.md

```diff
@@ -370,18 +370,36 @@ by feature with a [**custom script**][custom-script-mir-ext].
 
 > Adjust miRNAs' 'Name' attribute to account for the different genomic
 > locations the miRNA sequence is annotated on and ensure their uniqueness.
-> The name format is `SPECIES-mir-NAME-#` for pri-miRs, and 
-> `SPECIES-miR-NAME-#-ARM` or `SPECIES-miR-NAME-#` for mature miRNA with both
-> or just one arm respectively, where `#` is the replica integer. If a pri-miR
-> has a replica but its number is set in the 'ID' attribute, the first instance
-> does not has a suffix but the other one(s) do. If a precursor has no other
-> occurrences, no further modifications are made. On the other hand,
-> mature miRNA regions are extended on both sides to account for isomiR species
-> with shifted start and/or end positions without exceeding chromosome(s)
-> boundaries. If required, pri-miR loci are also extended to accommodate the
-> new miRNA coordinates. In addition, pri-miR names are modified to record the
-> final positions by appending `_-y` and `_+x` to them, where `y` is the 5'
-> shift and `x` the 3' shift.
+> Updates the attributes of precursor and mature miRNA entries to ensure
+> consistent naming based on their sequence variant and/or replica information.
```

Comment:
The term "replica" isn't standard terminology. Generally, "paralog" (paralogs are genes that arose through duplication within the genome of a single species) is the more conventional term for such situations, even if duplication is only _assumed_.

Whichever term you use, it will need a definition in the specific context. And just to be on the safe side, defining "sequence variations" in the specific context you are using it would be helpful as well.

Also, both would benefit from a concrete example.